test/integration/indexSpec.js

```diff
@@ -1,62 +1,232 @@
 // custom modules
 const db = require('../../dao/db');
 let server = require('../../index');
+const sqliteUtil = require('../util/SQLiteDB');
 
 //Require the dev-dependencies
 let chai = require('chai');
 let chaiHttp = require('chai-http');
-const sqliteDatabase = require('better-sqlite3');
 let should = chai.should();
 
 
 chai.use(chaiHttp);
 //Our parent block
 describe('All with queries', () => {
     before((done) => {
-        // Please look at the README for debugging to the console
-        const Testdb = new sqliteDatabase('airlinesTest.db', {memory: true});
-        Testdb.prepare('CREATE TABLE `tbl_airports` (`code` char(3) NOT NULL,`name` varchar(45) NOT NULL,PRIMARY KEY (`code`),UNIQUE (`code`));').run();
+        sqliteUtil.createSQLiteDatabase(function() {
+            // function indicating that this is function has finished
+            done();
```

Comment:
Just pass `done` to `createSQLiteDatabase` instead of making a trivial lambda wrapper.